backend/src/app.controller.ts

```diff
@@ -5,8 +5,12 @@ import { AppService } from './app.service';
 export class AppController {
   constructor(private readonly appService: AppService) {}
 
-  @Get()
-  getHello(): string {
-    return this.appService.getHello();
+  /**
+   * Healthcheck
```

Comment:
```suggestion
   * Ping
```

A health check is not typically done via a ping 😉